src/adapters/bidderFactory.js

```diff
@@ -235,6 +236,11 @@ export function newBidder(spec) {
         onBid: (bid) => {
           const bidRequest = bidRequestMap[bid.requestId];
           if (bidRequest) {
+            bid.adapterCode = adapterManager.getAdapterCode(bidRequest.bidder);
```

Comment:
this should default to `bidRequest.bidder` - if the request is for `pubmatic`, the adapter code should be `pubmatic`.